src/js/tracking.js

```diff
@@ -79,9 +79,6 @@ class Tracker {
     } else {
       data = this.currentPageView;
     }
-    if (this.initialized && this.trackingEnabled) {
-      ReactGA.pageview(data);
-    }
```

Comment:
based on https://github.com/codler/react-ga4#example this likely has to become:
```
ReactGA.send({ hitType: "pageview", page: data, title: "Some Title" });
```
in order for us to keep tracking pageviews...

Reply:
I understood that pageviews should be automatically tracked now, unless we do something custom:
https://developers.google.com/analytics/devguides/migration/measurement/virtual-pageviews#comparison_of_ua_and_ga4_virtual_pageviews
And (the _don't do this_ image) in https://medium.com/@vergil333/google-analytics-4-in-react-a250005c2455

Reply:
oh nice! I wasn't aware of this, then sorry about the noise 🙈